client/src/pages/CataloguePage/CataloguePage.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
+import CatalogMainSection from '../../components/CatalogMainSection/CatalogMainSection';
```

Comment:
we created an index.js file in the components folder where we export from it all our components. please adjust accordingly.

Reply:
done